src/content/docs/zh-cn/features/dialog.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Dialog
+description: 本机系统对话框，用于打开和保存文件，以及消息对话框。
+i18nReady: true
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+import PluginLinks from '@components/PluginLinks.astro';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+<PluginLinks plugin="dialog" />
+
+本机系统对话框，用于打开和保存文件，以及消息对话框。
+
+## 设置
+
+从安装对话框插件开始。
+
+<Tabs>
+    <TabItem label="自动">
+
+    使用项目的包管理器来添加依赖：
+
+    <CommandTabs npm="npm run tauri add dialog"
+    yarn="yarn run tauri add dialog"
+    pnpm="pnpm tauri add dialog"
+    cargo="cargo tauri add dialog" />
+
+    </TabItem>
+    <TabItem label="手动">
+
+    1. 运行 `cargo add tauri-plugin-dialog` 命令，将插件添加到项目的 `cargo .toml` 依赖中。
+
+    2. 修改 `lib.rs` 来初始化插件。
+
+    ```rust
+    // lib.rs
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Initialize the plugin
+            .plugin(tauri_plugin_dialog::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
+
+    3. 如果你想在JavaScript中创建对话框，还需要安装 npm 包：
+
+    <CommandTabs
+        npm="npm install @tauri-apps/plugin-dialog"
+        yarn="yarn add @tauri-apps/plugin-dialog"
+        pnpm="pnpm add @tauri-apps/plugin-dialog"
+    />
+
+</TabItem>
+</Tabs>
+
+## 用法
+
+对话框插件可以在 JavaScript 和 Rust 中使用。你可以这样使用它：
+
+在 JavaScript:
+
+- [创建 Yes/No 对话框](#创建-yesno-对话框)
+- [创建 Ok/Cancel 对话框](#创建-okcancel-对话框)
+- [创建 Message 对话框](#创建-message-对话框)
+- [打开一个文件选择对话框](#打开一个文件选择对话框)
+- [保存到文件对话框](#保存到文件对话框)
+
+在 Rust:
+
+- [建立一个询问对话框](#建立一个询问对话框)
+- [建立消息对话框](#建立消息对话框)
+- [建立一个文件选择器对话框](#建立一个文件选择器对话框)
+
+---
+
+### JavaScript
+
+可以在 JavaScript API 参考中查看所有 [Dialog 选项](/references/v2/js/dialog/)。
+
+{/* ASK */}
+
+#### 创建 Yes/No 对话框
+
+显示一个带有 “Yes” 和 "No" 按钮的提问对话框。
```

Comment:
```suggestion
显示一个带有 "Yes" 和 "No" 按钮的提问对话框。
```
Formatting nitpick. Let's stick to plain double quotation mark for now for easier consistency.